.babelrc

```diff
@@ -4,6 +4,7 @@
     ],
     "plugins": [
         "@babel/plugin-proposal-nullish-coalescing-operator",
-        "@babel/plugin-proposal-optional-chaining"
+        "@babel/plugin-proposal-optional-chaining",
+		["@babel/plugin-proposal-decorators", { "legacy": true }]
```

Comment:
I think this is not used, maybe a leftover from the earlier PR?

Reply:
This was actually not in the initial PR, but now of course I can't remember why I added it 😅 I'll double-check!